neuralforecast/models/kan.py

```diff
@@ -0,0 +1,435 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: ../../nbs/models.kan.ipynb.
+
+# %% auto 0
+__all__ = ['KANLinear', 'KAN']
+
+# %% ../../nbs/models.kan.ipynb 5
+from typing import Optional, Union
+
+import math
+
+import torch
+import torch.nn.functional as F
+
+from ..losses.pytorch import MAE
+from ..common._base_windows import BaseWindows
+
+# %% ../../nbs/models.kan.ipynb 6
+class KANLinear(torch.nn.Module):
```

Comment:
Maybe move KANLinear to common modules if we use it in two models? (if we intend to keep KANHITS?)

Reply:
Not keeping KANHITS, so I'll leave it here. Maybe we'll move it when we work on TKAN or TKAT.